locales/en.default.json

```diff
@@ -44,6 +44,7 @@
       "item_added": "Item added to your cart"
     },
     "share": {
+      "close": "Close share popup",
```

Comment:
@gretchen-willenberg Do you have any thoughts on what this text should be?

Reply:
I don’t think we have to include `popup` since that’s just the container vs the role/action. But, it would be good to align it to the initial state. What is that?

I’m guessing….
Action: I don’t like this literal action of the UI vs what user intent is
V1. `Open share` 
Role: 
v1. `Share product`
v2. `Share this product`

[Closing the popup](https://www.figma.com/file/JcYdswrk2JhqBTGBVWPM7J/Dawn-2.0?node-id=11628%3A105890) 
Role : 
v1. `Close`     
[See Polaris](https://polaris.shopify.com/content/actionable-language#section-close-vs-cancel) 
Action: 
V1. `Close share`


**Additional feedback:** Exclamation marks should be celebratory, and copying to clipboard doesn’t meet that threshold to me. Can we remove the ! And just say `Copied to clipboard`

**Side take:** Let's track the Shares for Blog post: My gut reaction is that end-user customers/readers are less likely to share a blog post before they read it? Under the title seems very immediate and gives users an action before reading the blog post…which to me is the primary action of visiting the page. I’m hesitant to drive attention away so quickly. On the other hand, it’s clearly secondary and not overly dominating. Use will tell us!